### ISSUE TITLE
Figure out why parameter descriptions aren't rendering

### ISSUE THREAD

**Issue by adriancole** [openzipkin/zipkin-api#11]
parameters like serviceName have a description, but the output doesn't show them

ex. http://zipkin.io/zipkin-api/#/paths/%252Ftraces doesn't render the description for serviceName

**Comment by adriancole** [comment]
updated to latest sway and got a warning.. maybe the schema is different than what we are using

```
$ npm test
> zipkin-api@0.0.0 test /Users/acole/oss/zipkin-api
> node validate.js

Warnings:
[{"code":"EXTRA_REFERENCE_PROPERTIES","message":"Extra JSON Reference properties will be ignored: description","path":["paths","/trace/{traceId}","get","responses","200","schema"]}]
Validation passed
```

**Comment by mohsen1** [comment]
That means a `$ref` has a sibling somewhere. All of `$ref` siblings will be overwritten with the result of the pointer by resolver.

**Comment by adriancole** [comment]
Thanks for the tip. maybe an upgrade fixes the field description thing.. we'll see https://github.com/openzipkin/zipkin-api/pull/12

**Comment by adriancole** [comment]
Fixed by #12
